tests/pytests/test_optimizer.py

```diff
@@ -162,8 +162,8 @@ def testOptimizer(env):
                     ['Type', 'INTERSECT', 'Counter', 10, 'Child iterators', [
                         ['Type', 'TAG', 'Term', 'foo', 'Counter', 14, 'Size', 10000],
                         ['Type', 'UNION', 'Query type', 'NUMERIC', 'Counter', 10, 'Child iterators', [
-                            ['Type', 'NUMERIC', 'Term', '6 - 12', 'Counter', 7, 'Size', 1600],
-                            ['Type', 'NUMERIC', 'Term', '14 - 50', 'Counter', 4, 'Size', 7600]]]]],
+                            ['Type', 'NUMERIC', 'Term', '0 - 10', 'Counter', 4, 'Size', 2400],
```

Comment:
why the swap here?

Reply:
The different tree logic caused the split to be different. By chance the 11 docs got divided this way